app/editor/package.json

```diff
@@ -1,129 +1,129 @@
 {
-    "name": "tno-editor-app",
-    "description": "Today's News Online Editor application",
-    "version": "0.1.0",
-    "license": "Apache-2.0",
-    "homepage": "/",
-    "packageManager": "yarn@3.2.0",
-    "dependencies": {
-        "@fortawesome/fontawesome-svg-core": "^6.1.1",
```

Comment:
@jdtoombs The app should have fixed versions otherwise when it gets installed in other environments it could be different than our local.